src/components/nodes/intermediateSignalCatchEvent/index.js

```diff
@@ -0,0 +1,20 @@
+import component from './intermediateSignalCatchEvent.vue';
+import merge from 'lodash/merge';
+import cloneDeep from 'lodash/cloneDeep';
+import intermediateMessageEventConfig from '@/components/nodes/intermediateMessageEvent';
+
+export default merge(cloneDeep(intermediateMessageEventConfig), {
```

Comment:
I guess this can clone intermediateSignalEvent when the other pr is merged. Or is it generic enough for it to be in intermediateEvent?

Reply:
Oops, this was a result of copy-paste. I should extend some base intermediate/signal event config. I can update this after your PR is merged.

Reply:
My tests are still running. So this could go in first and I update it.